docs/configuration/http_conn_man/route_config/rate_limits.rst

```diff
@@ -157,22 +157,28 @@ Header Value Match
   {
     "type": "header_value_match",
     "descriptor_value" : "...",
+    "expect_match" : "...",
     "headers" : []
   }
 
 
 descriptor_value
-    *(required, string)* The value to use in the descriptor entry.
+  *(required, string)* The value to use in the descriptor entry.
 
-:ref:`headers<config_http_conn_man_route_table_route_headers>`
-    *(required, array)* Specifies a set of headers that the rate limit action should match on. The
-    action will check the request's headers against all the specified headers in the config. A match
-    will happen if all the headers in the config are present in the request with the same values (or
-    based on presence if the ``value`` field is not in the config).
+expect_match
+  *(optional, boolean)* Specifies if the action should append a descriptor when it expects a match
```

Comment:
I think this sentence is redundant and a little confusing. The following two sentences explain very clearly what this value accomplishes.